qa/workunits/smb/tests/pytest.ini

```diff
@@ -0,0 +1,6 @@
+
+[pytest]
+addopts = -m 'default'
```

Comment:
Is this (`-m default`) added to be able to run the tests outside teuthology?

Reply:
I find the way pytest marks work a little quirky. This makes functions marked with default run even if you don't pass -m to the pytest cli command. It's just there to make sure the pytest script runs something as a backup.

Reply:
OK.